src/EllipsisNotation.jl

```diff
@@ -71,11 +71,6 @@ ArrayInterface.can_flatten(::Type{A}, ::Type{T}) where {A,T<:Ellipsis} = true
 end
 =#
 
-# avoid copying if indexing with .. alone, see
-# https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/issues/214
-@inline Base.getindex(A::AbstractArray, ::Ellipsis) = A
-@inline ArrayInterface.getindex(A, ::Ellipsis) = A
```

Comment:
The issue mentioned there is still open. We discussed streamlining the interpolation codes using `..`, but didn't implement it (as far as I remember). The only difference will be when calling `A[..]`. If it's on the left-hand side, it can subtly break (as fixed in this PR). If it's on the right-hand side, it's an indexing operation anyways and should be consistent in creating a copy of the data. So I think this should be fine.

Reply:
Do you have some specific tests in mind?

Reply:
What does it do without these dispatches? Does it make a copy?

Reply:
Yes, without these changes `A[..]` behaves as `A[:]`.